frontend/src/use-cases/create-account/CreateAccount.action-creator.jsx

```diff
@@ -12,33 +12,27 @@ import {
 } from "./CreateAccount.actions";
 
 import { redirectTo } from "../../app/views/gamma-redirect/GammaRedirect.view.action-creator";
-//import { toastOpen } from "../../app/views/gamma-toast/GammaToast.view.action-creator";
+import { toastOpen } from "../../app/views/gamma-toast/GammaToast.view.action-creator";
 
-export function createAccountValidateCid(data) {
+export function createAccountValidateCid(data, errorMsg) {
```

Comment:
Nice use of thunk action with side-effects, but what is toastOpen? :)

Reply:
This is a toast: https://material-ui.com/demos/snackbars/

It's opens a toast bar with the given error msg.

Reply:
Ok, but that component use internal state? 
https://github.com/cthit/Gamma/pull/56/files#diff-17ab056b33eda0c5af4228362614236dR97

Reply:
And the actions (at least from my opinion) is better in this form:

Reply:
It auto closes after a given duration to the toastOpen function. The only reason it should close is if you press the button on the toast button.